src/shared/Database/DatabaseMysql.h

```diff
@@ -27,6 +27,8 @@
 
 #ifdef _WIN32
 #include <mysql/mysql.h>
+#elif __APPLE__
```

Comment:
Why don't you just replace #ifdef and #elif with `#if defined(_WIN32) || defined(__APPLE__)` ?

Reply:
Why why is because i didn't know you could do that with preprocessor macros 😄